docs/docs/create-apps/general-app.md

```diff
@@ -14,50 +17,105 @@ Before deploying/launching your app with JHub Apps Launcher, you'll need:
+
+When a environment variables is added, users can then enter the key/value pair.
+
+![Create app form, environment variables section, one added](/img/create_app_env_var_2.png)
+
+JHub-apps allows the optional sharing of apps with others on the JupyterHub deployment. Users can 
+give either individuals or groups access to your apps. Additionally, users have the option to make
+an app fully open to the public - without authenticating via JupyterHub. All sharing options 
+should be set with care since from within the app, all users will have the permissions of the 
+original app creator. 
+
+![Create app form, sharing section](/img/create_app_sharing.png)
+
+Finally, users can select an image to use as a thumbnail for the app card on the home screen. If
+no thumbnail is selected, the card will display the logo of the framework in which it was
+created. 
+
+![Create app form, thumbnail section](/img/create_app_thumbnail.png)
+
+After submitting this form, users are redirected to the second stage of the process - server
+selection. Here, users must select the server/instance type (i.e. machines with CPU/RAM/GPU 
+resources) in which to deploy the app. This is only applicable if you're using 
+`KubeSpawner` as the spawner for JupyterHub.
+
+![Create app form, select server page](/img/create_app_select_server.png)
+
+Once you click "Deploy App", you'll be redirected to the app page which will automatically refresh
+when the app is ready for viewing. 
+
+### Details about each field are included below:
+
+| Field                                                           | Description  |
+| --------------------------------------------------------------- | ------------ |
+| Name                                                           | Provide a meaningful name for your application. |
+| Description (optional)                                         | Add addition information about the application. |
+| Framework                                                      | Select the framework used by your application from the dropdown. |
+| Software Environment                                           | Provide the environment used while developing your notebook/script which has `jhsingle-native-proxy`, the corresponding framework (e.g., `panel`, `gradio`, etc.), and additional requirements documented in framework-specific pages. |
+| Filepath                                                       | Provide the path (from root in JupyterLab) to your application's code. |
+| Keep alive                                                     | Toggle on to keep app alive indefinitely. If toggled off, app server will be shut down according to the `jupyter-idle-culler` timeout settings. |
+| Environment Variables (optional)                               | The set of environment variables, defined as valid JSON, to set before starting the app. This is useful when creating custom apps. |
```

Comment:
>The set of environment variables, defined as valid JSON, to set before starting the app. This is useful when creating custom apps.

We don't ask for a json anymore, since there is a form for key value pair.